source/css/_common/components/post/post-copyright.styl

```diff
@@ -4,8 +4,4 @@
   border-left: $post-copyright.border.width $post-copyright.border.style $post-copyright.border.color;
   background-color: $post-copyright.bg;
   list-style: none;
-
-  i.fa {
-    font-size: 15px;
```

Comment:
Screens with this changes: before and after?

Reply:
No changes are made here. `.fa` element has a style `font-size: 1em` in `font-awesome.css`, which means it will inherit the `font-size` from its parent element, so there's no need to define it again.

Reply:
Screens with before and after?

Reply:
Not sure. But I know that a icon with font-size:15px looks strange in a line of text with font-size:20px.